src/components/TeamBio.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import Image from 'react-bootstrap/Image'
+
+class TeamBio extends React.Component {
+    render() {
+        const { pic, name, role, bio } = this.props.tmData;
+
+        return (
+            <>
+                <Image src={ pic } fluid></Image>
+                <p>{ name }</p>
```

Comment:
Changing this to `h3 className="fs-5 my-2"` makes this look like a slightly larger, bold sub-heading with some extra space around it.